SUPPORTED_APIS.md

```diff
@@ -14,6 +14,7 @@ The `status` options are:
 
 | Namespace | API | <div style="width:130px">Status</div> | Description |
 | --- | --- | --- | --- |
+| `ANVIL` | `anvil_setMinGasPrice` | `NOT IMPLEMENTED` | Set the minimum gas price for the node. Unsupported by design |
```

Comment:
Maybe it makes sense to elaborate that this setting is only relevant for pre-EIP1559 chains?

Reply:
Done